README.md

```diff
@@ -178,6 +179,10 @@ If an issue _doesn't_ yet exist, please use these templates to create one:
 
 **[New action](https://github.com/PipedreamHQ/pipedream/issues/new?assignees=&labels=action%2C+enhancement%2C+good+first+issue%2C+help+wanted&template=action-request.md&title=%5BACTION%5D)**
 
+## Want to contribute?
+
+[Check out our contributing guide](https://github.com/PipedreamHQ/pipedream/blob/master/CONTRIBUTING.md) to learn how to involved in our community, learn how to build your own components, and more.
```

Comment:
"learn how to get involved"

Reply:
In the second clause you say "learn" again so maybe you can shorten the whole thing to:

> to learn how to get involved in our community, how to build your own components, and more.